src/groups/mqb/mqbstat/mqbstat_queuestats.cpp

```diff
@@ -651,6 +661,23 @@ void QueueStatsDomain::updateDomainAppIds(
     }
 }
 
+QueueStatsDomain::EventCb
+QueueStatsDomain::buildEventCallback(EventType::Enum type)
+{
+    // disambiguate mqbstat::QueueStatsDomain::onEvent
+    bdlf::MemFn<void (mqbstat::QueueStatsDomain::*)(
+        mqbstat::QueueStatsDomain::EventType::Enum,
+        bsls::Types::Int64,
+        const bsl::string&)>
+        f(&mqbstat::QueueStatsDomain::onEvent);
```

Comment:
This disambiguate step could be removed if I merge 2 overloads `onEvent(type,value)` and `onEvent(type,value,appId)` into one where `appId` is optionally specified: `onEvent(type,value,*appId = nullptr)`. In this case, null `appId` ptr means that we want to update "the entire queue" metrics, non-null means that we want to update metric per specific appId.

It is simple to merge these `onEvent` functions, but the downside of this merge is the fact that not all the metrics which make sense for the entire queue make sense per appId.